src/tasks/make-util.js

```diff
@@ -295,12 +295,12 @@ var ensureTool = function (name, versionArgs, validate) {
     if (versionArgs) {
         var result = exec(name + ' ' + versionArgs);
         if (typeof validate == 'string') {
-            if (result.output.trim() != validate) {
+            if (result.trim() != validate) {
```

Comment:
Examples on the [readme](https://github.com/shelljs/shelljs#execcommand--options--callback) make me think `output` should now be `stdout`. Does just trimming `result` do the same thing?

Reply:
Result was of type string in every case we used this